### DISCUSSION TITLE
Troubles with git-lfs v3

### DISCUSSION THREAD

**Discussion by gjasny** [Homebrew/discussions#2219] [Everyday usage]
Hello,

In Homebrew/homebrew-core#86082 `git-lfs` was upgraded from `v2.13.3` to `v3.0.1`. Unfortunately, not all git servers support the new git-lfs version. For example BitBucket only recently (7.17.0, 7.16.1: both unreleased) added support for version 3 (see [BSERV-12989](https://jira.atlassian.com/browse/BSERV-12989)).

As a local mitigation I added git-lfs `v2.13.3` to my personal tap: `gjasny/tap/git-lfs`. But maybe it might be beneficial for homebrew users to have a stable and widely supported git-lfs version available.

What would you prefer?
* Provide a separate git-lfs `v2` Formula in parallel to `v3`
* Revert the formula to `v2` and let some time pass before upgrading, again
* Do nothing

Thanks,
Gregor

PS: For issue discovery, the error was:
```
Error downloading object: foo.png (bc49efa): Smudge error: Error downloading foo.png (bc49efa25c6cde4f9b9e07ce0aace9b6d53a7c4ae273d2a64e9b5ffcb8ddb6df): batch request: Bitbucket cannot execute 'git-lfs-authenticate /~gregorj/lfs-test.git download'. The command is not supported as entered.
```

**Comment by carlocab** [comment]
If v2 satisfies the [requirements](https://docs.brew.sh/Versions#acceptable-versioned-formulae), we can add a formula for it. Otherwise, we'd rather it were maintained in external taps if people need it.

**Reply by gjasny** [reply]
Thanks. I filed https://github.com/Homebrew/homebrew-core/pull/86253

I consider this official formula an improvement over an external one because it will also provide a bottle which eases installation.
